### DISCUSSION TITLE
Button Loading Process

### DISCUSSION THREAD

**Discussion by untillnesss** [filamentphp/filament#8471] [Help] [answered]
When I use the Form from Resources, when carrying out a process, the button will automatically display the Loading effect as shown in the image below.

![image](https://github.com/filamentphp/filament/assets/33585758/93d48dac-6593-406c-9be8-fbc64b1886d5)

But when I use a Custom Form like the example below, the Loading effect doesn't appear.
Is there anything I missed?

![image](https://github.com/filamentphp/filament/assets/33585758/4886d4a0-0257-4afb-ada5-32025ab3e4e7)

**Comment by leandrocfe** [comment, marked as answer]
```html
<x-filament::button form="create" type="submit">
    Upload File
</x-filament::button>
```
